google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -198,37 +145,29 @@ public void exit(int exitCode) {
 
           if (deployOutput == null
               || deployOutput.getService() == null || deployOutput.getVersion() == null) {
-            consoleLogLn(
-                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
+            loggingHandler.print(
```

Comment:
I agree it's blurry.

On Fri, Jun 3, 2016 at 11:54 AM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/700#discussion_r65731108>
> :
>
> > @@ -198,37 +145,29 @@ public void exit(int exitCode) {
> >
> >            if (deployOutput == null
> >                || deployOutput.getService() == null || deployOutput.getVersion() == null) {
> > -            consoleLogLn(
> > -                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
> > +            loggingHandler.print(
>
> oh right. My thinking is that the console window sort of blurs the line
> between UI and logging output. It *can* be the output raw logs pipped to
> it. But its also the main / only form of certain information messages that
> the user get within the IDE when performing certain actions. But you are
> right that the rest of the console output (coming from the common lib) will
> not be localized.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/700/files/cf9d207bfecca398c61805abbbdf7ec498e8e5fc#r65731108>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5Hf6mnenzabcUb_zqu50XhTReK-d-ks5qIE5IgaJpZM4Im4aR>
> .
>